modules/olm-building-operator-catalog-image.adoc

```diff
@@ -32,16 +32,15 @@ footnoteref:[BZ1771329, The
 `oc adm catalog` command is currently only supported on Linux.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1771329[*BZ#1771329*])]
 endif::[]
-* `oc` version 4.3+
+* `oc` version 4.3.5+
 * `podman` version 1.5.1+
```

Comment:
@adellape Could you help update the version of the `podman` at the same time? Thanks! Details in: https://bugzilla.redhat.com/show_bug.cgi?id=1804690